WordPress/src/main/java/org/wordpress/android/datasets/SiteSettingsTable.java

```diff
@@ -29,6 +30,7 @@
     public static void createTable(SQLiteDatabase db) {
         if (db != null) {
             db.execSQL(SiteSettingsModel.CREATE_SETTINGS_TABLE_SQL);
+            db.execSQL(JetpackSettingsModel.CREATE_JP_SETTINGS_TABLE_SQL);
```

Comment:
> wouldn't it be better to do it in FluxC?

Yes, absolutely. But since that'd be a large undertaking and we haven't really planned for it we're going to continue in WPAndroid and migrate to FluxC whenever we get around to moving all of Site Settings over.